plugins/alerts/frontend/public/javascripts/countly.views.js

```diff
@@ -12,811 +12,483 @@
-            { value: 'New users', name: 'New users' },
-            { value: 'Total sessions', name: 'Total sessions' },
-            { value: 'Average session duration', name: 'Average session duration' },
-            { value: 'Bounce rate', name: 'Bounce rate (%)' },
-            { value: 'Number of page views', name: 'Number of page views' },
-            { value: 'Purchases', name: 'Purchases' },
-        ],
-        condition: [
-            { value: 'increased by at least', name: 'increased by at least' },
-            { value: 'decreased by at least', name: 'decreased by at least' },
-        ]
-    },
-    event: {
-        target: [],
-        condition: [
-            { value: 'increased by at least', name: 'increased by at least' },
-            { value: 'decreased by at least', name: 'decreased by at least' },
-        ]
-    },
-    crash: {
-        target: [
-            { value: 'Total crashes', name: 'Total crashes' },
-            { value: 'New crash occurence', name: 'New crash occurence' },
-            { value: 'None fatal crash per session', name: 'None fatal crash per session' },
-            { value: 'Fatal crash per session', name: 'Fatal crash per session' },
-        ],
-        condition: [
-            { value: 'increased by at least', name: 'increased by at least' },
-            { value: 'decreased by at least', name: 'decreased by at least' },
-        ]
-    },
-    rating: {
-        target: [
-            { value: 'Number of ratings', name: 'Number of ratings' },
-        ],
-        condition: [
-            { value: 'increased by at least', name: 'increased by at least' },
-            { value: 'decreased by at least', name: 'decreased by at least' },
-        ]
+var AlertDrawer = countlyVue.views.BaseView.extend({
+    template: '#alert-drawer',
+    mixins: [
+        countlyVue.container.dataMixin({
+            "externalDataTypeOptions": "/alerts/data-type",
+            "externalAlertDefine": "/alerts/data-define",
+        })
+    ],
+    components: {
     },
-    dataPoint: {
-        target: [
-            { value: 'Number of daily DP', name: 'Daily data points' },
-            { value: 'Hourly data points', name: 'Hourly data points' },
-            { value: 'Monthly data points', name: 'Monthly data points' }
-        ],
-        condition: [
-            { value: 'increased by at least', name: 'increased by at least' },
-            { value: 'decreased by at least', name: 'decreased by at least' },
-        ]
+    data: function () {
+        var appsSelectorOption = [];
+        for (var appId in countlyGlobal.apps) {
+            appsSelectorOption.push({label: countlyGlobal.apps[appId].name, value: appId});
+        }
+        var defaultAppsSelectorOption = Object.assign([], appsSelectorOption);
+        return {
+            title: "",
+            saveButtonLabel: "",
+            apps: [""],
+            defaultAppsSelectorOption,
+            appsSelectorOption,
```

Comment:
you need to set values for the variables - **defaultAppsSelectorOption** and **appsSelectorOption**
please try to avoid using es6 syntax in the frontend.